### ISSUE TITLE
Contribution guide?

### ISSUE THREAD

**Issue by astrosticks** [MaurycyLiebner/enve#263]
Hi! I love the promising features in Enve. I'd like to donate some time to making contributions. Is there a list anywhere of areas of development that need focus?

**Comment by AlexKiryanov** [comment]
Good evening! my name is Alexander) if I understand correctly, then you know c ++ and qt? if so, it would be great if you help) I can write what you can try to do.

**Comment by astrosticks** [comment]
Yes! It's been a while since I've touched either, but I am willing to get back into them. I've submitted C++ patches for Inkscape and Flameshot before.

**Comment by AlexKiryanov** [comment]
Good evening! It's great! Then let's start with these questions.

#241 
If you have any ideas on how to do this, then that would be great. But honestly, I myself do not yet know how to do this using code. If not. Then you can make a simpler option:
![scrollbar](https://user-images.githubusercontent.com/65244629/123701980-3ce9fd00-d86b-11eb-940f-0588b80eaa4c.jpg)

Probably it should be done here.
https://github.com/MaurycyLiebner/enve/blob/master/src/app/GUI/timelinewidget.cpp


and and this question #243 
Here the problem needs to be solved in this file (if I understood correctly)
The problem is that updating the value of the HEX field is not created when the object is selected. In getColorSetting function. 
https://github.com/MaurycyLiebner/enve/blob/master/src/app/GUI/ColorWidgets/colorsettingswidget.cpp
But I can be wrong and write complete nonsense, since I'm not a programmer, unfortunately (Look please, maybe you have the right thoughts about this.
